ui/page/debug_page.go

```diff
@@ -128,3 +138,19 @@ func (pg *DebugPage) Layout(gtx C) D {
 	}
 	return components.UniformPadding(gtx, container)
 }
+
+func (pg *DebugPage) resetDexData() {
+	// Show confirm modal before resetting dex client data.
+	confirmModal := modal.NewInfoModal(pg.Load).
+		Title("Confirm DEX Client Reset").
+		Body("You may need to restart godcr before you can use the DEX again. Proceed?").
+		NegativeButton(values.String(values.StrCancel), func() {}).
+		PositiveButton("Reset DEX Client", func() {
+			if pg.Dexc().Reset() {
```

Comment:
This method had a bug on dcrlibwallet which is fixed in a new commit. Will share the commit ref to use in go.mod shortly.